api/src/main/java/bio/terra/tanagra/service/databaseaccess/bigquery/BigQueryCellValue.java

```diff
@@ -0,0 +1,46 @@
+package bio.terra.tanagra.service.databaseaccess.bigquery;
+
+import bio.terra.tanagra.service.databaseaccess.CellValue;
+import bio.terra.tanagra.service.databaseaccess.ColumnSchema;
+import bio.terra.tanagra.service.search.DataType;
+import com.google.cloud.bigquery.FieldValue;
+
+/** A {@link CellValue} for BigQuery's {@link FieldValue}. */
+@SuppressWarnings("PMD.PreserveStackTrace")
```

Comment:
nit: can it be at the method? Is class the smallest granularity for this one?

Reply:
Yep, thanks!